repo/maintenance/maintenance_run.go

```diff
@@ -252,21 +252,26 @@ func Run(ctx context.Context, runParams RunParameters, safety SafetyParameters)
 }
 
 func runQuickMaintenance(ctx context.Context, runParams RunParameters, safety SafetyParameters) error {
+	s, err := GetSchedule(ctx, runParams.rep)
+	if err != nil {
+		return errors.Wrap(err, "unable to get schedule")
+	}
+
 	_, ok, emerr := runParams.rep.ContentManager().EpochManager(ctx)
 	if ok {
-		log(ctx).Debugf("quick maintenance not required for epoch manager")
+		log(ctx).Debugf("Epoch manager is available, run epoch maintenance only for quick maintenance")
```

Comment:
Are you planning to add a test for this case?

Reply:
Yes, I can add some UT once we decide the scope of this PR.
At present, there is no UT for the entire function of runQuickMaintenance, let me see how to add it.

Reply:
Tests added in #4185